cmd/machine-config-operator/bootstrap.go

```diff
@@ -64,6 +69,11 @@ func init() {
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.networkConfigFile, "network-config-file", "/assets/manifests/cluster-network-02-config.yml", "File containing network.config.openshift.io manifest.")
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.cloudConfigFile, "cloud-config-file", "", "File containing the config map that contains the cloud config for cloudprovider.")
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.proxyConfigFile, "proxy-config-file", "/assets/manifests/cluster-proxy-01-config.yaml", "File containing proxy.config.openshift.io manifest.")
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.keepalivedImage, "keepalived-image", "registry.svc.ci.openshift.org/openshift:keepalived-ipfailover", "Image for Keepalived.")
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.corednsImage, "coredns-image", "registry.svc.ci.openshift.org/openshift:coredns", "Image for CoreDNS.")
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.mdnsPublisherImage, "mdns-publisher-image", "registry.svc.ci.openshift.org/openshift:mdns-publisher", "Image for mdns-publisher.")
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.haproxyImage, "haproxy-image", "registry.svc.ci.openshift.org/openshift:haproxy-router", "Image for haproxy.")
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.baremetalRuntimeCfgImage, "baremetal-runtimecfg", "registry.svc.ci.oppenshift.org/openshift:baremetal-runtimecfg", "Image for baremetal-runtimecfg.")
```

Comment:
you could empty out the default value as well - no harm tho, you can do it after https://github.com/openshift/installer/pull/2025 as well (there are no templates for this _today_ in master)

Reply:
since I am changing these anyway, I will empty them out

Reply:
The flag for runtimecfg really should be "baremetal-runtimecfg-image" for consistency.

Reply:
fixed, thanks!